### ISSUE TITLE
Size does not change if set as shown in documentation

### ISSUE THREAD

**Issue by bmcockman** [biutas/ng-bootstrap-icons#5]
When assigning width/height via inline style or class to the component, the icon svg does not change size

**Comment by pablo-impactoai** [comment]
HTML
`<bi name="file-image" class="scaling-svg-container"></bi>`


CSS
```
.scaling-svg-container {
    position: relative; 
    height: 0; 
    width: 100px; 
    padding: 0;
    padding-bottom: 100px; 
    /* override this inline for aspect ratio other than square */
}

.scaling-svg-container::ng-deep svg {
    position: absolute; 
    height: 100%; 
    width: 100%; 
    left: 0; 
    top: 0;
}
```

**Comment by pablo-impactoai** [comment]
https://css-tricks.com/scale-svg/#option-4-use-the-padding-bottom-hack-on-a-container

**Comment by dailytabs** [comment]
Always 16x16 in a 24x24 box, not properly centered on either axis. Essentially unusable.

**Comment by biutas** [comment]
Fixed on 13.1.2-canary.1 version, now the size is defined by your application, either on a css class or directly on component.

BTW, now i'm following the angular tag, the 13.1.1 and 13.3.1 have bugs, so don't use them, go to 13.1.2-canary.1. This should be fixed on Angular 13.2.0.
